packages/phone-number-privacy/common/src/utils/logger.ts

```diff
@@ -30,7 +30,7 @@ export const rootLogger: Logger = createLogger({
 export function loggerMiddleware(req: Request, res: Response, next?: NextFunction): Logger {
   const requestLogger = rootLogger.child({
     endpoint: req.path,
-    sessionID: req.body.sessionID, // May be undefined
+    sessionID: req.body.sessionID || genSessionID(),
```

Comment:
Doesn't this screw up our request without sessionId counter? Imo i would also distinguish this from a requestId to distinguish invocations from the same session

Reply:
No it doesn't screw up the counter because the counter checks whether the request object has a sessionID, and this just adds one to the logger.

Reply:
Re your second point my reasoning here is that if no sessionID is provided by the client then a "session" is just one request, so a requestID and sessionID are the same thing if we add it on the server side like we do here. If I understand you correctly, then a "session" is different than a request in that a "session" may include retries and exist across services.